### ISSUE TITLE
[Bug] Google Colab: IndexError

### ISSUE THREAD

**Issue by arimanyus** [deezer/spleeter#323]
## Description

Upon running !spleeter separate -i h3RAwtNpQ46.mp3 -o /content, I received the following error:

> IndexError: index 1 is out of bounds for axis 1 with size 1. 
>  File "/usr/local/lib/python3.6/dist-packages/spleeter/separator.py", line 126, in stft
    dl, dr = (data[:, :, 0].T, data[:, :, 1].T) if inverse else (data[:, 0], data[:, 1])

## Step to reproduce

1. Received on Google Colab`
3. Got `IndexError` error

## Output

```bash
Traceback (most recent call last):
  File "/usr/local/bin/spleeter", line 8, in <module>
    sys.exit(entrypoint())
  File "/usr/local/lib/python3.6/dist-packages/spleeter/__main__.py", line 54, in entrypoint
    main(sys.argv)
  File "/usr/local/lib/python3.6/dist-packages/spleeter/__main__.py", line 46, in main
    entrypoint(arguments, params)
  File "/usr/local/lib/python3.6/dist-packages/spleeter/commands/separate.py", line 45, in entrypoint
    synchronous=False
  File "/usr/local/lib/python3.6/dist-packages/spleeter/separator.py", line 191, in separate_to_file
    sources = self.separate(waveform, audio_descriptor)
  File "/usr/local/lib/python3.6/dist-packages/spleeter/separator.py", line 157, in separate
    return self.separate_librosa(waveform, audio_descriptor)
  File "/usr/local/lib/python3.6/dist-packages/spleeter/separator.py", line 145, in separate_librosa
    stft = self.stft(waveform)
  File "/usr/local/lib/python3.6/dist-packages/spleeter/separator.py", line 126, in stft
    dl, dr = (data[:, :, 0].T, data[:, :, 1].T) if inverse else (data[:, 0], data[:, 1])
IndexError: index 1 is out of bounds for axis 1 with size 1
```

## Environment

|                   |                                 |
| ----------------- | ------------------------------- |
| OS                | Windows / Linux / MacOS / other |
| Installation type | Conda / pip / other             |
| RAM available     | XGo                             |
| Hardware spec     | GPU / CPU / etc ...             |

**Comment by mmoussallam** [comment]
Hi @arimanyus 

Is your file mono ? we had an issue with those that should be fixed in the latest version.

**Comment by arimanyus** [comment]
Hi @mmoussallam,

My file was indeed mono. I'll try it out once again and let you know.

**Comment by romi1502** [comment]
The fix was just pushed in version 1.5.1. Spleeter should now work (again :)) on mono files when the latest version is installed with pip (i.e. in the provided Google Colab).
